OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,511 @@
+
+## Data Delivery Metering & Fair Exchange
+
+We believe that this area of work is where the main crux for Decentralized Data Delivery Markets come to reality. Without it (i.e. without relying on a verified mediator to serve as escrow and referee), it will be impossible to have a fully permissionless, decentralized and open market for data delivery.
+
+#### How it is done traditionally 
+
+In traditional or Web 2.0 setups (e.g. CDNs, Mirrors, Static Servers and so on), the correct metering happens on the server side and it is done by the provider which the clients trust to do the right measurement. This measurement translates into a statement of how much the service got used and a respective invoice and request for payment is issued. This payment is facilitated by a trusted third party (e.g. credit card company) and a legal contract that can be used to resolve disputes in case the client fails to pay to the provider and/or the provider fails to deliver the service promised to the client. 
+
+In a decentralized environment that strives to be permissionless, we can’t expect to rely on such legal agreements and third parties and so, we essentially need a way to prove fair exchange. That is, we need to be able to prove that the service was delivered correctly and the metering of the provided service was done correctly. This will, in turn, verify the exchange between the two parties and issue the correct payment (i.e. fair exchange). 
+
+This trustless property is really important as we know that markets with no mediators, escrows and other assurance services (e.g. underground markets) are prone to scams, as the users have to put a lot of trust that the provider will execute on the agreement. In a trustless environment, once the transfer is done, there is no way to contest.
+
+#### Properties desired
+
+*   The exchanges of value MUST be verifiable and correct
+    *   Fairness: 
+        *   The payment MUST be fulfilled if the SLA is fulfilled
+        *   The payment for bandwidth/latency in token SHOULD match what has been agreed and fulfilled
+    *   Verifiability:
+        *   Both parties MUST be capable of verifying that the exchange was performed correctly
+        *   Bonus: Anyone SHOULD be able to verify that the exchange was performed correctly
+*   SHOULD be Trustless
+    *   Ideally, the network can perform the operations without having to leverage Trust
+    *   Nevertheless in many designs it might be needed to leverage reputation as a trust vehicle to guarantee high quality of service and/or reliability.
+
+#### How is Data Delivery Metering different from Fair Exchange
+
+The field of Fair Exchange overlaps in part with Service Metering. Some notable differences are:
+
+*   Fair Exchange can be used for non-digital goods also
```

Comment:
I'm unsure about this distinction. Metering can also be used for petrol 🤷 Not in this particular context, perhaps, but the same is true of fair exchange.